### ISSUE TITLE
UX Gotcha: Update Notice Modal Dialog

### ISSUE THREAD

**Issue by Saijin-Naib** [matteo-convertino/otpmanager-nextcloud#30]
In short, this blocks the user from getting access to their OTPs until you update. 

In my case, I had to login to Google Play again and answer with my OTP to update the app... bit the app locked me out until I updated.

**Comment by matteo-convertino** [comment]
Yes actually I should implement the fact that there may be some updates that may be optional and others not.

**Comment by Saijin-Naib** [comment]
Would locking it into a read-only state with a visual warning work? 

That way we can still read our OTPs as needed, and maybe it's less work for you to wire up mandatory vs optional updates?
